releases/SF30_Pro-SN30_Pro/1.34/sf30sn30pro.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2019 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.sf30sn30pro.firmware</id>
+  <name>SF30/SN30 Pro</name>
+  <summary>Firmware for the 8Bitdo SF30/SN30 Pro Game Controller</summary>
+  <description>
+    <p>
+      Updating the firmware on your SF30/SN30 Pro device improves performance and adds
+      new features.
+    </p>
+  </description>
+  <provides>
+    <!-- USB\VID_057E&PID_2009 -->
+    <firmware type="flashed">b3850dd5-a010-5d5c-8834-2ff809e95869</firmware>
+    <!-- USB\VID_2DC8&PID_5750 -->
+    <firmware type="flashed">b02f1319-c1c4-54f4-b7f2-5bb3552f6a9e</firmware>
```

Comment:
Look like it was intentionally removed (and don't work without it 🤷)
https://github.com/fwupd/8bitdo-firmware/commit/aba4713141312faeecf3bd8594579b58a3a17008